third-party/cmake/BuildLuarocks.cmake

```diff
@@ -37,29 +91,29 @@ endif()
 
 # Each target depends on the previous module, this serializes all calls to
 # luarocks since it is unhappy to be called in parallel.
-add_custom_command(OUTPUT ${DEPS_LIB_DIR}/luarocks/rocks/lua-messagepack
+add_custom_command(OUTPUT ${HOSTDEPS_LIB_DIR}/luarocks/rocks/lua-messagepack
```

Comment:
Yes, my idea was to run nvim.exe with wine but call the tests from the host - but I have not been very successful with this yet. If nvim crashes wine blocks travis :S